doc/changes/changes_5.0.0.md

```diff
@@ -0,0 +1,38 @@
+# Virtual Schema for MySQL 5.0.0, released 2024-01-15
+
+Code name: Changes related to Exasol V8 / importing text datatypes.
+
+## Summary
+
+The behaviour when it comes to character sets is now simplified,
+The target char set is now always UTF-8.
+The IMPORT_DATA_TYPES property (and value FROM_RESULT_SET) are now deprecated (change in vs-common-jdbc).
```

Comment:
Does this property still work or will it fail when trying to use it?

Reply:
It will throw an exception upon creation when you use the FROM_RESULT_SET value. You can see this in the changed tests.

Reply:
OK. When I read "Deprecated" I would have expected that the property still works but usage is discouraged. If It throws an exception I would explicitly mention it here, including the error message. So users know that they need to remove the property.

Reply:
That's how it's currently done.

Reply:
I've updated the summary.